config/release-45x-211.conf

```diff
@@ -0,0 +1,84 @@
+##### Defined in the script #####
+# CURRENT_DIR: execution dir
+#####
+
+##### local dirs #####
+# Unlikely to need modification
+#ZINC_DIR=${CURRENT_DIR}/zinc
+#SCALATEST_DIR=${SCRIPT_DIR}/scalatest
+#BUILD_DIR=/media/wpopielarski/Windows/Users/wpopielarski/work/scala-ide-target
+LOCAL_M2_REPO=/media/wpopielarski/Windows/Users/wpopielarski/.m2/repository
```

Comment:
These hardcoded paths need to be removed from the final version. I suggest that you symlink `$uber-build-dir/target` to somewhere else (`ln -s $uber-build-dir/target /somewhere/else`).